validator/src/main/resources/expected-data-template/dotnet/eks/linux/aws-sdk-call-trace.mustache

```diff
@@ -0,0 +1,68 @@
+[{
+  "name": "^{{serviceName}}$",
+  "http": {
+    "request": {
+      "url": "^{{endpoint}}/aws-sdk-call$",
+      "method": "^GET$"
+    },
+    "response": {
+      "status": "^500$"
+    }
```

Comment:
Remove all status checks, not relevant. We will follow up with other canaries for this change too

Reply:
Done